src/Octoshift/Services/ReclaimService.cs

```diff
@@ -163,6 +161,12 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                 continue;
             }
 
+            if (lines.Where(x => x.Contains(login)).Count() > 1 && lines.Where(x => x.Contains(userid)).Count() > 1)
+            {
+                _log.LogError($"Mannequin {login} is a duplicate. Skipping.");
+                continue;
+            }
```

Comment:
Doesn't this mean if there are duplicate rows it won't process any of them. Wouldn't you want it to process the first one and then skip all subsequent ones?

Reply:
That was my initial thought too but then I was worried the first one listed might not be correct. 

I.e. the first one is claimed by User1 but I actually wanted it claimed by User2 on the next row.